docs/advanced-node-scheduling.asciidoc

```diff
@@ -116,7 +116,7 @@ spec:
                 labelSelector:
                   matchLabels:
                     elasticsearch.k8s.elastic.co/cluster-name: quickstart
-                topologyKey: kubernetes.io/hostname
+              topologyKey: kubernetes.io/hostname
```

Comment:
I think you also need to remove 'weight: 100' in line 114 for this to be correct as per [spec](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/#podaffinityterm-v1-core).